src/Microsoft.IdentityModel.Tokens/JsonWebKey.cs

```diff
@@ -257,6 +263,13 @@ public bool HasPrivateKey
                     return false;
             }
         }
+
+        /// <summary>
+        /// Returns a <see cref="SignatureProvider"/> instance that will create signatures for this algorithm.
```

Comment:
change: 'instance that will create signatures for this algorithm." to instance that will provide signature support for this key and algorithm.